storehaus-core/src/main/scala/com/twitter/storehaus/FutureCollector.scala

```diff
@@ -19,24 +19,24 @@ package com.twitter.storehaus
 import com.twitter.util.Future
 
 /** A type to represent how Seq of futures are collected into a future of Seq[T] */
-trait FutureCollector[-T] extends java.io.Serializable {
-  def apply[T1<:T](in: Seq[Future[T1]]): Future[Seq[T1]]
+trait FutureCollector extends java.io.Serializable { self =>
```

Comment:
My bad. I was trying something else earlier and forgot to remove the self. Removed.